c_emulator/rvfi_dii.h

```diff
@@ -23,7 +23,222 @@ class rvfi_handler {
 private:
   void get_and_send_packet(packet_reader_fn reader, bool config_print);
 
+  void send_packet_raw(const void *data, size_t len, bool config_print);
   unsigned trace_version = 1;
   int dii_port = -1;
   int dii_sock = -1;
 };
+
+// ****************************************************************************
+
+typedef struct {
+  uint32_t rvfi_insn = 0; // 31 ..  0, Instruction word: 32-bit instruction
```

Comment:
The formatting of the trailing comments is a bit odd, maybe better to just place them before the field definition instead?